### PR TITLE
Add PPP skeleton for ptejasv

### PR DESCRIPTION
**Pull request by ptejasv**
# Changelog
- link page to PPP in AboutUs.md
- add general PPP structure to ptejasv.md

### CODE REVIEW

File: docs/team/ptejasv.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Paramita Tejasvi - Project Portfolio Page
+## Overview
```
Comment (Roycius) [inline docs/team/ptejasv.md:2]: Isn't this the overview of your contributions rather than that of the project?

**Review by Roycius — APPROVED** [review]
LGTM other than the potential issue pointed out.

File: docs/team/ptejasv.md (hunk 1 above)
Comment (ptejasv) [reply docs/team/ptejasv.md:2]: I think this refers to the project though

Screenshot 2021-11-04 at 3 32 57 PM

**Comment by richwill28** [comment]
Sorry for hijacking this PR @ptejasv, could you help us fix #171 as well?

File: docs/team/ptejasv.md (hunk 1 above)
Comment (Roycius) [reply docs/team/ptejasv.md:2]: Right. Sorry for the confusion.

**Comment by ptejasv** [comment]
> Sorry for hijacking this PR @ptejasv, could you help us fix #171 as well?

Okay! I've changed it in the last commit.

**Pull request merged**